StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/SavedPaymentMethodCollectionView.swift

```diff
@@ -361,6 +344,8 @@ extension SavedPaymentMethodCollectionView {
                     case .saved(let paymentMethod):
                         if let attributedText = attributedTextForLabel(paymentMethod: paymentMethod) {
                             label.attributedText = attributedText
+                        } else if let linkPaymentDetails = paymentMethod.linkPaymentDetails {
+                            label.text = linkPaymentDetails.formattedLast4
```

Comment:
Special case for Link payment methods in horizontal mode, where we want to show the last 4 instead of the display name due to space constraints.